### ISSUE TITLE
whitelist urls dont work beyond 1

### ISSUE THREAD

**Issue by rancer4** [kkapsner/CanvasBlocker#585]
I can add 1 url to the whitelist and then i try to add a second and it doesnt whitelist it. it will whitelist the first, i did what the information ! told me to do, seperate them with ,  a comma.  but, any after the first url doesn't whitelist.  i tried to use a trailing / forward slash at teh end of the url, i tried not, i tried instead of a , a comma, just entering it on the next line.  i tried adding a comma after the first urls and then putting the next url on the next line too.  i tried adding a , a comma and then a   space.  the nthe next url.
nothing but the first whitelist url in the url whitelist will whitelist any url.  

I dont know how to fix it.

Im using Firefox 64 bit 78.14 on windows 10.  on a pc........ ryzen 1700x

ive basically described how ive inputted my urls into the box for them.  and how only the first one will work and none after do. maybe a better descriptioin on how to do it or fixing the problem with why it doesnt work.  because, I have no clue.
basically right now and its not working, my url list looks like this 
https://newegg.com/,https://ebay.com/,https://amazon.com/
thanks

**Comment by rancer4** [comment]
oh ok thanks.  forgot about that thing.  didnt really know how to use it. what are the 4 different things stand for the 4 icons.  and if i make some permanent exception; is there one, is there a way to remove itlater in case i didnt like it.  sometimes i get bugged pages and i just want that gone so sure all ad makers could put their code through that function but im just trying to use the pages and make them functionable.

**Comment by kkapsner** [comment]
If you hover with your mouse over them they give a tooltip. But the red cross is to disable notifications, the check mark with the clock is for temporary whitelisting and the check mark with the looking glass is for inspecting the current whitelisting state.

Sure you can remove a whitelisting later in the settings.
